05-JWT-Basics/starter/errors/index.js

```diff
@@ -0,0 +1,9 @@
+const CustomAPIError = require('./custom-error')
+const BadRequestError = require('./bad-request')
+const UnauthenticatedError = require('./unauthenticated')
+
+module.export = {
```

Comment:
```suggestion
module.exports = {
```
This needs to be `exports`